streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -1334,4 +1680,19 @@ private AssignmentInfo checkAssignment(final Set<String> expectedTopics,
 
         return info;
     }
+
+    private void assertEquivalentAssignment(final Map<String, List<TaskId>> thisAssignment,
```

Comment:
Can we just rely on the equals function of `map` and `list` here? I think it checks for "exact equality", while here `otherAssignment` being a super-set of thisAssignment can still pass.

Reply:
Well we also check for the size to be the same, but this could be improved I guess. I'm not sure we can use the map equals because the values are not necessarily equal until sorted.

But we could sort them and then use list equality instead.

Reply:
SG.